qt/scientific_interfaces/Direct/ALFView.cpp

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "ALFView_model.h"
+#include "ALFView_presenter.h"
+
+// will need these later
+#include "MantidQtWidgets/Common/FunctionBrowser.h"
+#include "MantidQtWidgets/InstrumentView/InstrumentWidget.h"
+#include "MantidQtWidgets/InstrumentView/InstrumentWidgetPickTab.h"
+#include "MantidQtWidgets/Plotting/PreviewPlot.h"
+
+#include "ALFView.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+using namespace Mantid;
```

Comment:
I think this is superfluous and it's good practice to try and avoid bringing in entire namespaces.